src/engine/BMGame.php

```diff
@@ -1882,13 +1882,13 @@ public function set_turbo_sizes(array $turboSizeArray) {
             $die = $this->playerArray[$playerIdx]->activeDieArray[$dieIdx];
 
             if (!$die->has_skill('Turbo')) {
-                $this->message = 'Cannot set turbo size for a non-turbo die';
-                return FALSE;
+                // ignore any turbo values supplied for non-turbo dice
```

Comment:
Can you say more about why this change is needed?  I don't think random_ai was submitting invalid turbo values?